conf/test_krakenuniq.config

```diff
@@ -0,0 +1,33 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test for
+    metagenomics krakenuniq.
+
+    Use as follows:
+        nextflow run nf-core/eager -profile test_krakenuniq,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'KrakenUniq test profile'
+    config_profile_description = 'Minimal test dataset to check the metagenomics krakenuniq pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input = 'https://github.com/nf-core/test-datasets/raw/eager/testdata/Mammoth/samplesheet_v3.tsv'
+
+    // Genome references
+    fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/eager/reference/Mammoth/Mammoth_MT_Krause.fasta'
+
+    // Metagenomics
+    run_metagenomics                = true
+    metagenomics_profiling_tool     = 'krakenuniq'
+    metagenomics_profiling_database = 'https://github.com/nf-core/test-datasets/raw/taxprofiler/data/database/krakenuniq/testdb-krakenuniq.tar.gz'
```

Comment:
We should make a proper test data for eager (so we actually gets hits), and make sure it's on the eager branch too.

We cannot guaruntee stability of the taxprofiler database (might be updated to match more recent version than what we have in eager, for example)

Reply:
For comment above, I described what the input is to the existing MALT/Kraken2 eager test databases here: https://github.com/nf-core/test-datasets/tree/eager?tab=readme-ov-file#databases

Reply:
For current PR: updated test dataset to eager branch version.

I will open an issue to extend test datasets to also include 'hits', rather than just testing for functionality of the subworkflow